tests/Http/Api/ActivityApiTest.php

```diff
@@ -86,14 +78,13 @@ public function testActivityIndexWithCampaignRunIdQuery()
     {
         $signup = factory(Signup::class)->create();
 
-        $this->json('GET', $this->activityApiUrl . '?filter[campaign_run_id]=' . $signup->campaign_run_id);
+        $this->get('api/v2/activity?filter[campaign_run_id]=' . $signup->campaign_run_id);
```

Comment:
Looks great, Just wondering why you prefer hard coding the url instead storing in a prop?